client/rpc/client_test.go

```diff
@@ -147,7 +147,7 @@ func testPermissions(t *testing.T,
 	nonceString := ""
 
 	_, err := Permissions(nodeClient, keyClient, publicKeyString, addressString,
-		nonceString, "set_base", []string{permAddressString, "root", "true"})
+		nonceString, "setBase", []string{permAddressString, "root", "true"})
```

Comment:
There are extensive unit tests in `/manager/eris-mint/state/permissions_test.go` that have been passing without failure